commons/src/main/kotlin/com/simplemobiletools/commons/compose/theme/Theme.kt

```diff
@@ -65,13 +67,17 @@ internal fun Theme(
         updateRecentsAppIcon(baseConfig, context)
     }
 
+    val dimensions = if (configuration.screenWidthDp <= 600) CommonDimensions else Sw600DpDimensions
```

Comment:
It's better to approach it with https://developer.android.com/reference/kotlin/androidx/compose/material3/windowsizeclass/WindowSizeClass

Reply:
Yeah, not sure why I did this. I wanted to keep it the same as previously, but I guess anything other than compact would be the same as this. Anyways, I think I will remove that other class for now, since it is exactly the same as `CommonDimensions` for now.